WooCommerce/Classes/ViewModels/CardPresentPayments/CardPresentModalNonRetryableError.swift

```diff
@@ -40,13 +43,15 @@ final class CardPresentModalNonRetryableError: CardPresentPaymentsModalViewModel
         return topTitle + bottomTitle
     }
 
-    init(amount: String, error: Error) {
+    init(amount: String, error: Error, onDismission: @escaping () -> Void) {
```

Comment:
nit: maybe `onDismiss` to be consistent with other simialr cases in the codebase?
```suggestion
    init(amount: String, error: Error, onDismiss: @escaping () -> Void) {
```

Reply:
Great point! changed :)